file-formats/smbc/type/zif_aff_smbc_v1.intf.json

```diff
@@ -0,0 +1,57 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "SMBC AFF Type",
+    "originalLanguage": "en"
+  },
+  "descriptions": {
+    "types": [
+      {
+        "name": "TY_CREATION_MODE",
```

Comment:
Why does the AFF Handler for interfaces only serialize the descriptions of types in system YI3?
@schneidermic0

Reply:
See the ABAP Doc comments classified with `class="shorttext synchronized"`. For these a description in the metadata is created. If you use only `class="shorttext"` no description is created.

Does this answer your question?

Reply:
yes, thanks